src/controllers/loginController.js

```diff
@@ -0,0 +1,72 @@
+let passport = require('passport')
+
+let db = require('../db')
+let ApiError = require('../lib/apiErrors')
+let $ = require('../lib/controllerHelpers')
```

Comment:
It was intentionally a $. Since I'm essentially defining a library of helper functions that will be used a lot in every controller, I was going for something similar to the idea of jQuery where it's super accessible without much typing, but it wouldn't make much of a difference to just change the name upon importation. So, I'll swap it out.